events/guides/WIX-STORES-EVENTS/AddToCart.md

```diff
@@ -18,22 +18,42 @@ Triggered when user clicks on the ‘Add to cart’ button.
 |type|text|Product type: physical or digital|
 |isPremium|boolean|Whether the Wix site is Premium|
 |userId|text|User ID|
-|metasiteId|text|Wix site ID|
+|metasiteId|text|Wix site ID| -->
 
-**Example**:
-```JSON
+| Name |  Type  | Description |
+| --- | --- | --- |
+| origin  | string  | Wix App that emitted this event, such as "Stores", "Bookings" |
+| id | GUID  | Product ID  |
+| name  | string  | Product name  |
+| price  | number | Product price |
+| currency  | currency  | Default site currency in ISO-4217 format  |
+| quantity  | number  | Product quantity  |
+| sku  | string  | Product Stock Keeping Unit (SKU)  |
+| type  |  string | Product type (for example, "physical") |
```

Comment:
"physical" and "digital"

Reply:
OK. To be clear - are these the only two possible values? No other values are possible? Because this is the description I am providing: 
Product type. Possible values are `physical` and `digital`.